### DISCUSSION TITLE
[10주차] 온라인 스터디

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#73] [General]
~ 20:15 각자 PR리뷰
~ 21:00 발표
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 발표
1. 유채린
 * [Network] OSI 7 Layer #70
2. 이주형
 * [Spring] Spring mvc #72
3. 김정윤
 * [Network] Cookie와 Session #68

**Comment by ChaerinYu** [comment]
### 회고 (a.k.a. 출석체크)
* 참석 확인용이므로 `감사합니다!` 만 작성해주셔도 좋아요 😊
* 좋았던 점
* 건의사항
* 느낀점

**Reply by KJY97** [reply]
감사합니다!

**Reply by leeejuhyeong** [reply]
다음주부터는 OS 들어갑니다..

**Reply by ChaerinYu** [reply]
오늘도 참여해주셔서 감사합니다 🙂
